### ISSUE TITLE
Dimension teleport bug.

### ISSUE THREAD

**Issue by CaptainStalfoss** [FossilsArcheologyRevival/FossilsArcheologyRevival#1256]
I think i found a bug, But not sure how relevant it is.

Basically when i teleport to the "treasure room" dimension, Its a black void, 
(witch i kind of expected, that isn't the issue.)

The problem is, when i teleport back to the Overworld. 
i get stuck in a void and the world wont ever load. 
reloading save wont recover ether, basically this bricks the entire world.

Now i'm sure the cause for this is because i teleported there via Mod commands.
But i figure its worth mentioning.
Even if the bug only happens when you "cheat", the risk of hard locking the world is worth looking into.

SHORT VERSION:
Teleporting via mods to treasure room Dim, and then back to the Overworld, locks the Overworld in a inescapable void.

**Comment by Alex-the-666** [comment]
fixed
